### PR TITLE
Office hardening update

### PR DESCRIPTION
**Pull request by SamBH**
Update to Client Device design to update and expand guidance in alignment with ACSC hardening guide for Microsoft Office.

### CODE REVIEW

**Comment by zedzedzed** [comment]
Hi Sam - I was trying to resync this PR with the latest from develop but it pulled in a lot more than desired so have tried to backtrack. Could you please verify this PR includes everything you were expecting?

**Comment by SamBH** [comment]
All looks good to me, believe it is ready to commit.

File: blueprint/client-devices.md [hunk 1]
```diff
@@ -1401,6 +1401,89 @@ Decision Point | Design Decision | Justification
...
+  * Block Win32 API calls from Office macro<br>92E97FA1-2EDF-4476-BDD6-9DD0B4DDDC7B
+  * Block Office communication application from creating child processes<br>26190899-1602-49E8-8B27-EB1D0A1CE869.
+* Flash content
+  * Justification: To align with the ACSC Microsoft Office hardening guidance.
+  * Block Flash activation in Office documents: Block all activation
+* Loading external content
+  * Justification: To align with the ACSC Microsoft Office hardening guidance.
+  * Always prevent untrusted Microsoft Query files from opening: Enabled
+  * Don’t allow Dynamic Data Exchange (DDE) server launch in Excel: Enabled
+  * Don’t allow Dynamic Data Exchange (DDE) server lookup in Excel: Enabled
```
Comment (zedzedzed) [inline blueprint/client-devices.md]: This line is in here twice. Is it missing an option or just duplicated?

**Comment by SamBH** [comment]
Believe this is just a duplicate.

Sam Brazier-Hollins
Managing Consultant
www.oobe.com.au<https://aus01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.oobe.com.au%2F&data=04%7C01%7C%7C28e25c8a2f4746eb508b08d937803450%7C05f72b90815c4cb98b4096c29b862189%7C0%7C0%7C637601846075445925%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=iAKRFEoJysGq5ANGoKCk5IbhY2VqDN1khNlN9q6fj28%3D&reserved=0>
t: +61 2 6162 1447 | m: +61 409 903 349

***@***.***

**Review by zedzedzed — APPROVED** [review]

**Pull request merged**